Rivet/RivetSamples/Complete-Migration.ps1

```diff
@@ -0,0 +1,21 @@
+
+function Complete-Migration
+{
+    [CmdletBinding()]
+    [Rivet.Plugin([Rivet.Events]::AfterMigrationLoad)]
+    param(
+        [Parameter(Mandatory)]
+        # The migration the operation is part of.
+        [Rivet.Migration] $Migration
+    )
+
+    Set-StrictMode -Version 'Latest'
+    Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
+    
+    $problems = $false 
+
+    if( $problems )

```

Comment:
I'm not sure how this example is supposed to work. How does `$problems` get assigned?

Reply:
This is just a Rivet Sample for the new `AfterMigrationLoad` Rivet Event. I didn't want to put some random validation in here so I just left it bare bones. I've added a comment into this file now to show where validation should be happening.